main.py

```diff
@@ -18,6 +18,15 @@
 LOGIN_URL = ""
 INVITE_LINK = ""
 
+@asynccontextmanager
+async def on_startup(app: FastAPI):
+    await api.setup()
+    await db.setup()
+
+    yield
+
+    await api.close()
+    # Hier kann noch selbst eine Methode, die je nach Datenbank variiert, hinzugefügt werden, um die Datenbank zu "schließen"
 
 app = FastAPI()
```

Comment:
`on_startup` wird aktuell noch nicht aufgerufen
```suggestion
app = FastAPI(lifespan=on_startup)
```